lib/scala/kg/src/main/scala/stores/kg/neo4j/Neo4jKgStoreTransactionWrapper.scala

```diff
@@ -112,6 +112,48 @@ class Neo4jKgStoreTransactionWrapper(configuration: Neo4jStoreConfiguration, tra
       s"MATCH (node:Node) RETURN ${nodePropertyNamesString}, rand() as rand ORDER BY rand ASC LIMIT 1"
     ).toNodes.head
 
+  /**
+   * Get top edges using pageRank grouped by relation that have the given node ID as an object
+   */
+  final override def getTopEdgesByObject(limit: Int, objectNodeId: String): List[KgEdge] = {
+    transaction.run(
+      s"""
+         |MATCH (subject:Node)-[edge]->(object:Node {id: $$objectNodeId})
+         |WHERE type(edge)<>"PATH"
```

Comment:
Put the magic string in a constant, like the SourceRelationship constant. I should have done this with val PathRelationship = "PATH" before.